### ISSUE TITLE
Panic not managed when sub key is null

### ISSUE THREAD

**Issue by sebbalex** [italia/publiccode-parser-go#11]
This issue seems related to #10 but this behave differently.

Using a publiccode as in the following examples ad unmanaged exception is thrown, this happens when a subkey is `null`

**json example**
```json
"intendedAudience": {
    "countries": null,
    "unsupportedCountries": ["gb"]
  }
```

**yaml example**
```yaml
intendedAudience:
  countries: 
  unsupportedCountries:
  - gb
```

**Panic**
``` 
2019/07/29 15:00:10 http: panic serving [::1]:55075: key "IntendedAudience/Countries" is empty. Remove it or fill with valid values
goroutine 9 [running]:
net/http.(*conn).serve.func1(0xc0000aa780)
	/usr/local/Cellar/go/1.12.6/libexec/src/net/http/server.go:1769 +0x139
panic(0x1360b40, 0xc000b55d50)
	/usr/local/Cellar/go/1.12.6/libexec/src/runtime/panic.go:522 +0x1b5
github.com/italia/publiccode-parser-go.(*Parser).decoderec(0xc00009e700, 0xc0005e57b0, 0x10, 0xc000b567e0, 0x1, 0xc000b55d30, 0x0)
	/Users/sebbalex/go/src/github.com/italia/publiccode-parser-go/parser.go:165 +0x13dc
github.com/italia/publiccode-parser-go.(*Parser).decoderec(0xc00009e700, 0x0, 0x0, 0xc000b56720, 0x0, 0x135dea0, 0xc000b56630)
	/Users/sebbalex/go/src/github.com/italia/publiccode-parser-go/parser.go:160 +0xec3
github.com/italia/publiccode-parser-go.(*Parser).Parse(0xc00009e700, 0xc0009f2a80, 0x8c2, 0xa80, 0x0, 0xc0000863d0)
	/Users/sebbalex/go/src/github.com/italia/publiccode-parser-go/parser.go:52 +0x229
main.parse(0xc0009f2a80, 0x8c2, 0xa80, 0xc0009f2a80, 0x8c2)
	/Users/sebbalex/workspace/go-test/src/main.go:55 +0x54
main.pcPayLoad(0x144e600, 0xc0004da000, 0xc0004f2400)
	/Users/sebbalex/workspace/go-test/src/main.go:107 +0x543
net/http.HandlerFunc.ServeHTTP(0x13e7e90, 0x144e600, 0xc0004da000, 0xc0004f2400)
	/usr/local/Cellar/go/1.12.6/libexec/src/net/http/server.go:1995 +0x44
github.com/gorilla/mux.(*Router).ServeHTTP(0xc0004d8000, 0x144e600, 0xc0004da000, 0xc0004f2200)
	/Users/sebbalex/go/src/github.com/gorilla/mux/mux.go:210 +0xe3
net/http.serverHandler.ServeHTTP(0xc0000b8a90, 0x144e600, 0xc0004da000, 0xc0004f2200)
	/usr/local/Cellar/go/1.12.6/libexec/src/net/http/server.go:2774 +0xa8
net/http.(*conn).serve(0xc0000aa780, 0x144f240, 0xc00009c340)
	/usr/local/Cellar/go/1.12.6/libexec/src/net/http/server.go:1878 +0x851
created by net/http.(*Server).Serve
	/usr/local/Cellar/go/1.12.6/libexec/src/net/http/server.go:2884 +0x2f4
```

**Comment by ododa** [comment]
Ciao @sebbalex. I tried to reproduce this failure using the example you provided but to no avail. Could it be that the issue has been sorted in the meantime?
This is the output I got:

`validation ko:`
`intendedAudience/countries: invalid type <nil>.`

**Comment by sebbalex** [comment]
HI @ododa it is very likely fixed in meantime, but it would be great having some tests on this matter

**Comment by bfabio** [comment]
Fixed in 2.0.0.
